src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Todo.css";
+
+const Todo = props => {
+    return (
+        <div 
+            className={`task${props.completed ? " completed" : ""}`}
```

Comment:
your bug for strike through is here....you are missing a hyphen (-) before completed...replace the space with a hyphen and it should work.